src/core/iomgr/udp_server.c

```diff
@@ -235,8 +235,10 @@ static int prepare_socket(int fd, const struct sockaddr *addr, int addr_len) {
   rc = setsockopt(fd, IPPROTO_IP, IP_PKTINFO, &get_local_ip,
                   sizeof(get_local_ip));
   if (rc == 0 && addr->sa_family == AF_INET6) {
+#if !TARGET_OS_IPHONE
```

Comment:
Then you'd probably need at least a defined(__APPLE__) also, right?

On Mon, Aug 24, 2015 at 4:51 PM Michael Lumish <notifications@github.com>
wrote:

> In src/core/iomgr/udp_server.c
> <https://github.com/grpc/grpc/pull/3049#discussion_r37818914>:
>
> > @@ -235,8 +235,10 @@ static int prepare_socket(int fd, const struct sockaddr *addr, int addr_len) {
> >    rc = setsockopt(fd, IPPROTO_IP, IP_PKTINFO, &get_local_ip,
> >                    sizeof(get_local_ip));
> >    if (rc == 0 && addr->sa_family == AF_INET6) {
> > +#if !TARGET_OS_IPHONE
>
> port_platform.h checks it with #if TARGET_OS_IPHONE. I just negated that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/3049/files#r37818914>.
>